data/usercontext.css

```diff
@@ -97,76 +91,65 @@
-}
-
-.tabbrowser-tab[usercontextid]:hover .tab-background-start:not([selected="true"]) {
-  background-image: linear-gradient(to left, var(--identity-tab-color) 0%, var(--identity-tab-color) 70%, transparent 80%, transparent 100%);
-}
-
-.tabbrowser-tab[usercontextid] .tab-background-start[selected="true"]::before {
-  background-image: linear-gradient(to left, var(--identity-tab-color) 0%, var(--identity-tab-color) 30%, transparent 100%),
-                    url(chrome://browser/skin/tabbrowser/tab-selected-start.svg) !important;
-  background-size: var(--identity-stroke-background-size), 100% 100% !important;
-  background-position: 0px 28px, 0;
-  background-repeat: no-repeat;
-}
-
-.tabbrowser-tab[usercontextid] .tab-background-end:not([selected="true"]) {
-  background-image: linear-gradient(to right, var(--identity-tab-color) 0%, var(--identity-tab-color) 50%, transparent 50%, transparent 100%);
-  background-position: 0 28px;
-  background-size: var(--identity-stroke-background-size);
-  background-repeat: no-repeat;
-}
-
-.tabbrowser-tab[usercontextid][beforehovered=true] .tab-background-end:not([selected="true"]) {
-  background-image: linear-gradient(to right, var(--identity-tab-color) 0%, var(--identity-tab-color) 22%, transparent 30%, transparent 100%);
-}
-
-.tabbrowser-tab[usercontextid]:hover .tab-background-end:not([selected="true"]) {
-  background-image: linear-gradient(to right, var(--identity-tab-color) 0%, var(--identity-tab-color) 70%, transparent 80%, transparent 100%);
-}
-
-.tabbrowser-tab[usercontextid] .tab-background-end[selected="true"]::before {
-  background-image: linear-gradient(to right, var(--identity-tab-color) 0%, var(--identity-tab-color) 30%, transparent 100%),
-                    url(chrome://browser/skin/tabbrowser/tab-selected-end.svg) !important;
-  background-size: var(--identity-stroke-background-size), 100% 100% !important;
-  background-position: 0 28px, 0;
-  background-repeat: no-repeat;
-}
-
-.tabbrowser-tab[usercontextid] .tab-background-middle {
-  background-position: 0 28px, 0;
-  background-repeat: repeat-x;
-  background-size: var(--identity-stroke-background-size), auto;
-  background-image: linear-gradient(to right, var(--identity-tab-color) 0%, var(--identity-tab-color) 100%), none;
-}
-
-.tabbrowser-tab[usercontextid] .tab-background-middle[selected="true"] {
-  background-image: url(chrome://browser/skin/tabbrowser/tab-active-middle.png),
-                    linear-gradient(to right, var(--identity-tab-color) 0%, var(--identity-tab-color) 100%),
-                    none;
-  background-position: 0, 0 28px, 0;
-  background-size: var(--tab-stroke-background-size), var(--identity-stroke-background-size), auto;
-  background-repeat: repeat-x;
+/* reset nightly containers */
+.tabbrowser-tab[usercontextid] {
+  background-image: none;
+  background-size: none
```

Comment:
Missing semi-colon here.
Make all of these "!important" or they are not removing my native styling in Linux at least.